daemon/cmd/fqdn.go

```diff
@@ -470,13 +470,15 @@ func (d *Daemon) notifyOnDNSMsg(lookupTime time.Time, ep *endpoint.Endpoint, epI
 			} else if serverSecID, exists := ipcache.IPIdentityCache.LookupByIP(serverIP); exists {
 				secID := d.identityAllocator.LookupIdentityByID(d.ctx, serverSecID.ID)
 				// TODO: handle IPv6
-				lr.LogRecord.DestinationEndpoint = accesslog.EndpointInfo{
-					IPv4: serverIP,
-					// IPv6:         serverEP.GetIPv6Address(),
-					Labels:       secID.Labels.GetModel(),
-					LabelsSHA256: secID.GetLabelsSHA256(),
-					Identity:     uint64(serverSecID.ID.Uint32()),
-					Port:         uint16(serverPort),
+				if secID != nil {
```

Comment:
I presume this means that we will still send a log message, but it will be missing destination endpoint information.

I wonder whether instead we should just try to do a check earlier in the function for whether the identity cache is initialized, and drop the entire notification if the daemon is not ready?

@jrajahalme any thoughts on this?

Reply:
I presume this can happen now due to DNS proxy being initialized earlier in bootstrap, hence this should also be marked as `Fixes: #12964` and backported to 1.7., 1.8, and 1.9.

I'm fine by doing the access log, but leaving out the info we don't have yet. To that end I'd prefer setting the fields not dependent on `secID` and leaving only those out if `secID` is `nil`.